pytorch_pfn_extras/training/extension.py

```diff
@@ -128,32 +128,27 @@ def load_state_dict(self, to_load):
 
 class _CallableExtension(Extension):
```

Comment:
I think now it's better to name it as:
```suggestion
class _WrappedExtension(Extension):
```